Sources/LiveKit/Participant/LocalParticipant.swift

```diff
@@ -307,7 +307,7 @@ public class LocalParticipant: Participant {
     @objc
     public func publish(data: Data,
```

Comment:
can we rename this to `publishData`, too, so that it's easier to refer to the same API in the docs?